rebar.config

```diff
@@ -11,13 +11,13 @@
 
 {project_plugins, [covertool, rebar3_ex_doc, {rebar3_hank, "~> 0.3.0"}]}.
 
-{deps, [{hex_core, "0.8.4"}, {verl, "1.1.1"}]}.
+{deps, [{hex_core, "0.10.1"}, {verl, "1.1.1"}]}.
 
 {profiles, [
     {test, [
         {extra_src_dirs, ["test/support"]},
         {overrides, [{override, rebar3,[{deps, [{erlware_commons, "1.3.1"}]}]}]},
-        {deps, [{hex_core, "0.8.4"},
+        {deps, [{hex_core, "0.10.1"},
```

Comment:
I'd get rid of this one. Doesn't `rebar3` just accumulate deps in profiles (i.e. concatenate them with whatever's in the "default" profile)?

Reply:
Agreed, this should be possible to take out.